js/start.md

```diff
@@ -277,7 +277,6 @@ $ react-native init myReactNativeApp
 $ cd myReactNativeApp
 $ npm install --save aws-amplify
 $ npm install --save aws-amplify-react-native
-$ react-native link
```

Comment:
You are right, it should `react-native link amazon-cognito-identity-js`

Reply:
I'm not sure this is correct. `react-native link` command should link any/all native modules that are installed. installing aws-amplify will install the dependency of amazon-cognito-identity-js which exposes a native module and coincidentally should be linked here.